net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/dictionary

```diff
@@ -0,0 +1,2 @@
+#ATTRIBUTE       Daily-Session-Time      3000    integer
```

Comment:
Reserved for later use? ;)

Reply:
No, the OPNsense doc wants this value, but it's already defined by mods-enabled/counter. Perhaps because this howto is for Linux. I just want to let it there if there will be some questions in the future :)